src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,23 @@
+@mixin on-phone {
+  @media (min-width: 488px) {
+    .container {
+      --colum-count: 2;
+    }
+  }
+}
+
+@mixin on-tablet {
+  @media (min-width: 768px) {
+    .container {
+      --colum-count: 3;
```

Comment:
The same typo is present here. The correct variable name should be `--column-count`.